src/classes/UTIL_OrgTelemetry_SObject_BATCH_TEST.cls

```diff
@@ -519,69 +447,77 @@ private class UTIL_OrgTelemetry_SObject_BATCH_TEST {
     }
 
     /**
-     * @description Builds test opportunity records with the given field values
-     * @param numOpps: number of opportunities to build
-     * @param fieldValues: map of values by SObjectField to set on the opportunities
-     * @return List of uninserted Opportunities
-     */
-    private static List<Opportunity> buildOpps(Integer numOpps, Map<Schema.SObjectField, Object> fieldValues) {
-        List<Opportunity> opps = new List<Opportunity>();
-
-        for (Integer i = 0; i < numOpps; i++) {
-            Opportunity opp = new Opportunity(
-                Name = 'Test Opp ' + i,
-                StageName = UTIL_UnitTestData_TEST.getClosedWonStage(),
-                CloseDate = Date.today()
-            );
-
-            for (Schema.SObjectField field : fieldValues.keySet()) {
-                opp.put(field, fieldValues.get(field));
-            }
+    * @description Insert Opportunities without parent Rd
+    * @param size amount of opps to be inserted
```

Comment:
`* @description Creates Opportunities not related to a Recurring Donation`
`* @param size Number of Opportunities to create`

Similar is for the next `createOpps()` ApexDoc.